bedrock/firefox/templates/firefox/geolocation.html

```diff
@@ -201,6 +201,18 @@ <h3 data-accordion-role="tab">{{ _('How do I turn off Location-Aware Browsing pe
   </div>
 </section>
 
+<section>
+  <h3 data-accordion-role="tab">{{ _('How can I add support for Geolocation to my web site?') }}</h3>
+  <div data-accordion-role="tabpanel">
+    <p>
+    {% trans url="https://developer.mozilla.org/en/Using_geolocation" %}
+    Web developers can integrate the Geolocation support into your service by
+    following the instructions at <a href="{{ url }}" title="developer.mozilla.org | Using Geolocation">Mozilla Developer Center Using Geolocation</a>.
```

Comment:
Nit: I don't think the `title` attribute here adds any extra meaning over the link text, so it can probably be removed